packages/protocol/contracts/libs/LibSharedConfig.sol

```diff
@@ -16,20 +16,25 @@ library LibSharedConfig {
         return
             TaikoData.Config({
                 chainId: 167,
-                maxNumBlocks: 2049, // up to 2048 pending blocks
-                blockHashHistory: 100000,
-                // This number is calculated from maxNumBlocks to make
-                // the 'the maximum value of the multiplier' close to 20.0
+                // Assuming proof time 1 hour, block time 10 seconds
+                // Level of parallelization is 60*60/10 = 360
```

Comment:
If the proof time is 1 hour, then a block will need `proofTime` (1 hour) + `UncleProofDelay (state.avgProofTime ≈ 1 hour)` ≈ 2 hours before getting verified?